apps/widget-e2e/src/describer/modelAssessment/whatIfCounterfactuals/describeSubBarChart.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { BarChart } from "../../../util/BarChart";
+import { ScatterChart } from "../../../util/ScatterChart";
+import { IModelAssessmentData } from "../IModelAssessmentData";
+
+export function describeSubBarChart(dataShape: IModelAssessmentData): void {
+  const props = {
+    chart: undefined as unknown as ScatterChart,
```

Comment:
how is the different from `ScatterChart | undefined`?

Reply:
I just checked locally, these doesn't work in same way. Here undefined is assigned as the value for chart rather than Type.